$NetBSD: patch-ac,v 1.8 2010/12/03 12:22:19 wiz Exp $

Avoid using obsolete texi2html command line flags.
Fix installation using the "install-sh" the package.

--- Makefile.orig	2009-05-28 22:13:17.000000000 +0000
+++ Makefile
@@ -80,7 +80,7 @@ documentation: $(addprefix doc/, ffmpeg-
                                  ffplay-doc.html general.html $(ALLMANPAGES))
 
 doc/%.html: doc/%.texi
-	texi2html -monolithic -number $<
+	texi2html -monolithic -number-sections $<
 	mv $(@F) $@
 
 doc/%.pod: doc/%-doc.texi
@@ -93,15 +93,15 @@ install: $(INSTALL_TARGETS-yes)
 
 install-progs: $(PROGS) $(INSTALL_PROGS_TARGETS-yes)
 	install -d "$(BINDIR)"
-	install -c -m 755 $(PROGS) "$(BINDIR)"
+	install -c -c -m 755 $(PROGS) "$(BINDIR)"
 
 install-data: $(DATA_FILES)
 	install -d "$(DATADIR)"
-	install -m 644 $(DATA_FILES) "$(DATADIR)"
+	install -c -m 644 $(DATA_FILES) "$(DATADIR)"
 
 install-man: $(MANPAGES)
 	install -d "$(MANDIR)/man1"
-	install -m 644 $(MANPAGES) "$(MANDIR)/man1"
+	install -c -m 644 $(MANPAGES) "$(MANDIR)/man1"
 
 uninstall: uninstall-progs uninstall-data uninstall-man
 
